### DISCUSSION TITLE
Copilot not working with Data Spell: "An error occurred while fetching the completions. Please try again."

### DISCUSSION THREAD

**Discussion by Snimm** [community/community#32938] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/53926889/189958994-f60ecbc6-8d62-4396-adb6-885bb50a12ae.png)

I just installed Data Spell and Copilot is not working with it. No matter how many times I try again, it does not work. It always says " An error occurred while fetching the completion, please try again later" 

I have no other downloaded plugin installed.   I am on Windows 10 64 bit. 

Although I found someone having a similar problem with Pycharm ,  https://github.com/community/community/discussions/14537 
my Copilot is working fine with Pycharm.

**Comment by iamcjwilliams** [comment]
Hello @Snimm Our engineering team is looking into this issue

**Reply by tilmann1** [reply]
@Snimm This looks like a network configuration issue (possibly proxy-related?). Can you please file a ticket under https://support.github.com/contact/bug-report so GitHub Support can help find the root cause? Thanks!
